projects/ngx-d3timeline/src/lib/axis/line.ts

```diff
@@ -7,31 +8,21 @@ export interface Line {
   y2: number;
 }
 
-export function createLine(
-  x1: number,
-  y1: number,
+export function createAxisAlignedLine(
```

Comment:
Axis is abstract in this context
```suggestion
export function createOrientedLine(
```